egs/wsj/s5/steps/nnet3/chain/build_tree.sh

```diff
@@ -85,19 +85,10 @@ echo $nj >$dir/num_jobs
 [[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj || exit 1;
 
 # Set up features.
-
-if [ -f $alidir/final.mat ]; then feat_type=lda; else feat_type=delta; fi
-echo "$0: feature type is $feat_type"
+echo "$0: feature type is delta"
```

Comment:
sorry-- you should actually leave this file as it was.  it uses the features from a baseline GMM-based system, which will normally be LDA.